.nvmrc

```diff
@@ -0,0 +1 @@
+8
```

Comment:
Can you explain this please? We will use it in CLI. I thought it should be `4`?

Reply:
I just copied this from the CLI. It doesn't really need to be 4 unless you want some extra assurances that it will work on 4, but Travis testing matrix should catch it anyway